src/app/shared/components/template/services/template.service.ts

```diff
@@ -38,7 +38,7 @@ export class TemplateService {
       await this.initialiseDefaultFieldAndGlobals();
     });
     // Update default values when skin changed to allow for skin-specific global overrides
-    this.skinService.currentSkin$.subscribe(async (skin) => {
+    this.skinService.activeSkin$.subscribe(async (skin) => {
```

Comment:
nit(non-blocking)
Another question of awareness, would it make more sense for the skin service to re-initialise the default field and globals instead of the template service listening for the config change? I'm on the fence myself, but part of me thinks better to have more logic in the skin service as it can actually check whether we need to reinitialise before calling the method on the template service